cmd/zqd/README.md

```diff
@@ -0,0 +1,11 @@
+# `zqd`
+
+`zqd` serves a REST-like API used to manage and query archives of log data
+. It is used as the backend service for the [Brim](https://github.com/brimsec/brim)
```

Comment:
```suggestion
`zqd` serves a REST-like API used to manage and query "spaces" that contain
log data. It is used as the backend service for the [Brim](https://github.com/brimsec/brim)
```

1. When the period was on a separate line, it was being rendered with an extra space char.

2. The way this was originally written, it made it sound like zqd might be _only_ for archives. While writing the suggestion, I struggled a bit coming up with wording, since it begs the question of what we'd call the smaller collections of logs that aren't official zar-managed archives. So I've opted to use our term "spaces" since I figure this way developers that find their way to zqd after using the Brim app will see the connection. It also reinforces that the space metadata is important to differentiating zqd-managed logs vs. "just a directory full of log files" like zq can do at the CLI.